src/main/resources/v2/Java/ApiClient.mustache

```diff
@@ -103,6 +103,7 @@ public class ApiClient {
     authentications = new HashMap<String, Authentication>();{{#authMethods}}{{#is this 'basic'}}
     authentications.put("{{name}}", new HttpBasicAuth());{{/is}}{{#is this 'api-key'}}
     authentications.put("{{name}}", new ApiKeyAuth({{#is this 'key-in-header'}}"header"{{/is}}{{#isNot this 'key-in-header'}}"query"{{/isNot}}, "{{keyParamName}}"));{{/is}}{{#is this 'oauth'}}
+    authentications.put("{{name}}", new OAuth());{{/is}}{{#is this 'bearer'}}
```

Comment:
Now this is fun, since there is also a security scheme type for OAuth in particular. I would expect that to generate an OAuth authentication. Getting an OAuth authentication when one wanted a plain bearer based authentication might be confusing.

Reply:
Right, basically just picked the `OAuth` class because it add the `Authentication` header which we're looking for. I guess to avoid confuse we could create another class. But i'm open to ideas if you have something in mind.

Thanks again for your time to check this.